src/components/LabelTitle/index.js

```diff
@@ -5,47 +5,29 @@ import PropTypes from "prop-types"
 import styles from "./styles.module.css"
 
 const LabelTitle = props => {
-    return (
-        <div className={styles.labelTitle}>
-            <h1 className={styles.headerTitle}>Data Nutrition Label</h1>
-            <div className={styles.pageSubHeaderRow}>
-                <h2 className={styles.datasetLabelHeader}>
-                    {props.datasetName}
-                </h2>
-                <a className={styles.iconLink} href={props.datasetNameLink}>
-                    <img
-                        className={styles.iconLinkImg}
-                        src={require("../../images/linkimg.png")}
-                        alt="icon for link"
-                        id="datasetLink"
-                    />
-                </a>
-            </div>
-            <div className={styles.link}>
-                <p className={styles.datasetOriginLink}>{props.datasetOrg}</p>
-            </div>
-            <div className={styles.link}>
-                <a className={styles.datasetLink} href={props.datasetOrgLink}>
-                    <img src="/linkimg.png" alt="" id="datasetLink" />
-                </a>
-            </div>
-            <span className={styles.weightedLine}> </span>
-        </div>
-    )
+  return (
+    <div className={styles.labelTitle}>
+      <h1 className={styles.headerTitle}>{props.datasetName}</h1>
+      <div className={styles.pageSubHeaderRow}></div>
+
+      <p className={styles.datasetOriginLink}>Dataset Nutrition Label</p>
```

Comment:
I think that, even though the mocks say "Dataset Nutrition Label" here, we eventually want this to be the name of the org, not our name. I think we should reinsert `props.datasetOrg` here.